Resources/Textures/_DV/Objects/Specific/Chemistry/pills_canister.rsi/meta.json

```diff
@@ -0,0 +1,33 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Inhands taken from cev-eris at https://github.com/discordia-space/CEV-Eris/blob/2b969adc2dfd3e9621bf3597c5cbffeb3ac8c9f0/icons/obj/chemical.dmi. Pill canister taken from Taken from /tg/ at https://github.com/tgstation/tgstation/blob/master/icons/obj/medical/chemical.dmi on 20/12/2024.",
```

Comment:
permalink not master + date
![06:47:21](https://github.com/user-attachments/assets/b06689b2-71b9-4fed-abf1-bf8d3afa7c17)